pkg/processor/build/builder.go

```diff
@@ -1363,16 +1363,52 @@ func (b *Builder) commandsToDirectives(commands []string) (map[string][]function
 	currentDirective := "preCopy"
 
 	// iterate over commands
-	for _, command := range commands {
+	for i := 0; i < len(commands); i++ {
+		command := commands[i]
+
 		if strings.TrimSpace(command) == "@nuclio.postCopy" {
 			currentDirective = "postCopy"
 			continue
 		}
 
+		// check if the last character is backslash. if so treat this and the next command as one multi-line command
+		// example: commands = []{"echo 1\", "2\", "3"} -> "RUN echo 1 2 3"
+		aggregatedCommand := ""
+		for {
+
+			if strings.TrimSpace(command) == "@nuclio.postCopy" {
+				currentDirective = "postCopy"
+				break
+
+			} else if len(command) != 0 && command[len(command)-1] == '\\' {
+
+				// remove whitespace
+				command = command[:len(command)-1]
+
+				// add command to the aggregated multi-line command
+				aggregatedCommand += command
+
+				if len(commands) > i + 1 {
```

Comment:
switch between the if\else order by
- using `<=` and break
- removing indentation for `i++` & `command = commads[i]`